src/main/java/dukchess/entity/Event.java

```diff
@@ -0,0 +1,34 @@
+package dukchess.entity;
+
+public class Event extends Task {
+    private String at;
```

Comment:
I would recommend a more descriptive name, as `at` could refer to time or place.

Reply:
Thanks for reviewing! These are useful feedback and I'll edit the code accordingly.